zebra/zserv.c

```diff
@@ -524,6 +554,9 @@ static void zserv_process_messages(struct event *thread)
 	/* Reschedule ourselves if necessary */
 	if (need_resched)
 		zserv_event(client, ZSERV_PROCESS_MESSAGES);
+
+	if (client_ibuf_fifo_cnt == p2p)
```

Comment:
still not clear about this condition.
I think it'd be fine to trigger the read event if there's some room in the queue, the incoming fifo. I don't think we have to be more specific, at this time.

Reply:
Uploaded new review based on our discussion